addon/components/o-s-s/select.ts

```diff
@@ -98,18 +98,7 @@ export default class OSSSelect extends BaseDropdown<OSSSelectArgs> {
       return;
     }
 
-    scheduleOnce('afterRender', this, () => {
-      const referenceTarget = this.container.querySelector('.upf-input');
-      const floatingTarget = document.querySelector(`#${this.portalId}`);
-
-      if (referenceTarget && floatingTarget) {
-        this.cleanupDrodpownAutoplacement = attachDropdown(
-          referenceTarget as HTMLElement,
-          floatingTarget as HTMLElement,
-          { placementStrategy: 'auto' }
-        );
-      }
-    });
+    scheduleOnce('afterRender', this, this.setupDropdownAutoplacement);
```

Comment:
Move to private method to prevent the following warning:
![Screenshot 2025-05-06 at 15 24 25](https://github.com/user-attachments/assets/bc2acb73-4da9-4499-bb06-35e079fd56d8)